Thuku/bucketlist-api
# Ft bucket items

#### What does this PR do?
- Test various parts of the code and endpoints.
- Allow creation of bucketlists and their activities
#### Description of Task to be completed?
- Create, update, view and delete a bucket list
- Add, update, view or delete items in a bucket list

#### How should this be manually tested?
 -  Run the following command to tests
  `nosetests --with-coverage --cover-package=app/ && coverage report`
- Check the API documentation on how to test the endpoints
[http://docs.bucketlist19.apiary.io/](url)
